src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -117,6 +119,20 @@ class GrabbitConfiguration {
         )
     }
 
+    /**
+     * Sorts the {@code pathConfigurations} on the basis of path depth
```

Comment:
Hello @jbornemann 
for handling this case, I've removed '/.' from the end before sorting the pathConfigurations lexicographically. Added a spec for clarification as well.
Please suggest if this would work, and also, if there are other edge cases may be, that needs to be taken care of.

Thanks